content/docs/install/access/index.md

```diff
@@ -0,0 +1,330 @@
+    ```
+
+    The Keptn API endpoint (either via the internal or external IP; try both if unsure) is: `http://${INTERNAL_NODE_IP}:${API_PORT}/api` or `http://${EXTERNAL_NODE_IP}:${API_PORT}/api`
+
+    *Optional:* Store Keptn API endpoint in an environment variable.
+
+    For Linux and Mac:
+    ```console
+    KEPTN_ENDPOINT=http://${EXTERNAL_NODE_IP}:${API_PORT}/api
+    ```
+
+    For Windows:
+    ```console
+    $Env:KEPTN_ENDPOINT = 'http://${EXTERNAL_NODE_IP}:${API_PORT}/api'
+    ```
+
+:warning: **Warning:** If you do not set up TLS encryption, all your traffic to and from the Keptn endpoint is not encrypted.
+
+### Option 3: Expose Keptn via an Ingress
+
+1. **Install Keptn:** For installing Keptn on your cluster, please use the Keptn CLI.
+Depending on whether you would like to install the execution plane for continuous delivery, add the flag `continuous-delivery.enabled=true`.
+  ```console
+  helm install keptn keptn/keptn -n keptn --version=$KeptnVersion --create-namespace --set=continuous-delivery.enabled=true
+  ```
+
+1. **Install an Ingress-Controller and create an Ingress:**
+  Please first install your favorite Ingress-Controller and then apply an Ingress object in the `keptn` namespace, 
+  which points to the service `api-gateway-nginx` on port 80. Note that the [Kubernetes Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) allows to setup TLS encryption. **Note**: Using Openshift 3.11 requires to use a configuration for this platform.
+
+    Commonly used Ingress-Controller are e.g. Istio and NGINX:
+
+    <details><summary>**Istio 1.8+** (recommended for use-case continuous delivery)</summary>
+    <p>
+
+    * Istio provides an Ingress Controller. To install Istio, please refer to the [official documentation](https://istio.io/latest/docs/setup/install/).
+
+    * [Determine the ingress IP](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports):
+
+      ```console
+    kubectl -n istio-system get svc istio-ingressgateway
+      ```
+
+    * Create an `ingress-manifest.yaml` manifest for an Ingress object in which you set IP-ADDRESS or your hostname and then apply the manifest. (**Note:** In the example below, `nip.io` is used as wildcard DNS for the IP address.)
+
+      ```yaml
+    apiVersion: networking.k8s.io/v1
+    kind: Ingress
+    metadata:
+      annotations:
+        kubernetes.io/ingress.class: istio
+      name: api-keptn-ingress
+      namespace: keptn
+    spec:
+      rules:
+      - host: <IP-ADDRESS>.nip.io
+        http:
+          paths:
+          - path: /
+            pathType: Prefix
+            backend:
+              service:
+                name: api-gateway-nginx
+                port:
+                  number: 80
+      ```
+
+      ```console
+    kubectl apply -f ingress-manifest.yaml
+      ```
```

Comment:
(this comment is meant for this whole ingress section here)
We do provide an optional ingress resource directly in the Keptn helm chart.
See: https://github.com/keptn/keptn/blob/master/installer/manifests/keptn/values.yaml#L415-L423

Maybe this could be changed in a follow-up task?